src/components/LoginPage/index.js

```diff
@@ -42,18 +56,45 @@ class LoginPage extends React.Component {
       password
     };
 
-    this.setState({
-      loading: true
-    });
+    if (!email || !password) {
+      // if user tries to submit empty email/password
+      this.setState({
+        error: 'Please enter your email and password'
+      });
+    } else {
+      if (this.validateEmail(email)) {
+        this.setState({
+          loading: true
+        });
 
-    axios
-      .post(`${API_BASE_URL}/users/login`, userCredentials)
-      .then((res) => {
-        if (res.data.status === 'success') {
-          this.setState({
-            loading: false
-          });
+        axios
+          .post(`${API_BASE_URL}/users/login`, userCredentials)
+          .then((res) => {
+            if (res.data.status === 'success') {
+              this.setState({
+                loading: false
+              });
+              console.log('Login successful...');
```

Comment:
@MutegekiHenry kindly remove the console log functions from your PR please.

Reply:
Okay then